openslides/core/static/templates/index.html

```diff
@@ -9,6 +9,7 @@
 <link rel="stylesheet" href="static/css/openslides-libs.css">
 <link rel="stylesheet" href="static/css/app.css">
 <script src="static/js/openslides-libs.js"></script>
+<script src="static/ckeditor/ckeditor.js"></script>
```

Comment:
Is there a special reason why the ckeditor can not be part of openslides-libs.js?